terraform/modules/vpc/vpc.tf

```
@@ -1,33 +1,37 @@
 module "vpc" {
-  source                                     = "dwp/vpc/aws"
-  version                                    = "2.0.14"
-  vpc_name                                   = "ci-cd"
-  region                                     = data.aws_region.current.name
-  vpc_cidr_block                             = var.vpc_cidr_block
-  interface_vpce_source_security_group_count = length(var.vpc_endpoint_source_sg_ids)
-  interface_vpce_source_security_group_ids   = var.vpc_endpoint_source_sg_ids
-  interface_vpce_subnet_ids                  = aws_subnet.private.*.id
-  gateway_vpce_route_table_ids               = aws_route_table.private.*.id
-  ec2autoscaling_endpoint                    = true
-  ec2_endpoint                               = true
-  ec2messages_endpoint                       = true
-  kms_endpoint                               = true
-  logs_endpoint                              = true
-  monitoring_endpoint                        = true
-  s3_endpoint                                = true
-  ssm_endpoint                               = true
-  ssmmessages_endpoint                       = true
-  secretsmanager_endpoint                    = true
-  ecrapi_endpoint                            = true
-  ecrdkr_endpoint                            = true
-  ecs_endpoint                               = true
-  elasticloadbalancing_endpoint              = true
-  events_endpoint                            = true
-  application_autoscaling_endpoint           = true
-  kinesis_firehose_endpoint                  = true
-  glue_endpoint                              = true
-  emr_endpoint                               = true
-  dynamodb_endpoint                          = true
-  efs_endpoint                               = true
-  common_tags                                = merge(var.tags, { Name = var.name })
+  source                                   = "dwp/vpc/aws"
+  version                                  = "3.0.5"
+  vpc_name                                 = "ci-cd"
+  region                                   = data.aws_region.current.name
+  vpc_cidr_block                           = var.vpc_cidr_block
+  interface_vpce_source_security_group_ids = var.vpc_endpoint_source_sg_ids
+  interface_vpce_subnet_ids                = aws_subnet.private.*.id
+  gateway_vpce_route_table_ids             = aws_route_table.private.*.id
+
+  aws_vpce_services = [
+    "ec2autoscaling",
+    "ec2",
+    "ec2messages",
+    "kms",
+    "logs",
+    "monitoring",
+    "s3",
+    "ssm",
+    "ssmmessages",
+    "secretsmanager",
+    "ecrapi",
+    "ecr.dkr",
+    "ecs",
+    "elasticloadbalancing",
+    "events",
+    "application_autoscaling",
+    "kinesis_firehose",
+    "glue",
+    "emr",
+    "dynamodb",
+    "efs",
+    "sns",
+    "athena"
```

would be slightly better to have a comma here
```suggestion
    "athena",
```